src/openrct2-ui/interface/Graph.cpp

```diff
@@ -248,13 +248,12 @@ static void graph_draw_hovered_value(
     {
         return;
     }
-
-    gfx_draw_dashed_line(dpi, info.coords.x, chartFrame.GetTop(), info.coords.x, info.coords.y, DEFAULT_DASHED_LENGTH, 0);
-    gfx_draw_dashed_line(dpi, chartFrame.GetLeft() - 10, info.coords.y, info.coords.x, info.coords.y, DEFAULT_DASHED_LENGTH, 0);
+    gfx_draw_dashed_line(dpi, ScreenLine(ScreenCoordsXY(info.coords.x, chartFrame.GetTop()), info.coords), DEFAULT_DASHED_LENGTH, 0);
```

Comment:
If it works, use the following short notation
```suggestion
    gfx_draw_dashed_line(dpi, {{ info.coords.x, chartFrame.GetTop() }, info.coords }, DEFAULT_DASHED_LENGTH, 0);
```